packages/liveblocks-core/src/types/ai.ts

```diff
@@ -603,3 +592,32 @@ export function patchContentWithDelta(
       return assertNever(delta, "Unhandled case");
   }
 }
+
+/**
+ * Creates a receiving tool invocation part for testing purposes.
+ * This helper eliminates the need to manually create fake tool invocation objects
+ * and provides a clean API for tests.
+ */
+export function hydrateReceivingToolInvocation(
```

Comment:
The term "hydrate" isn't obvious here to me, is that a common use of it? And since it's purely test-related, could it be a test utility (not in a source file) like other `createDummy*` utilities?